docs/fundamentals/systems/overlays/configuration.md

```diff
@@ -220,6 +220,153 @@ export const hidesOnOutsideClick = () => {
 };
 ```
 
+## contentElementToFocus
+
+In case of an sticky element inside an overlay with a lot of content you can set `contentElementToFocus` to make sure the scrollable element gets focused.
+
+```js preview-story
+import { css, html, LitElement } from 'lit';
+import { uuid } from '@lion/ui/core.js';
+import { OverlayMixin } from '@lion/ui/overlays.js';
+
+class MyOverlayFrame extends LitElement {
+  static get styles() {
+    return css`
+      :host {
+        display: flex;
+        flex-direction: column;
+        max-width: 600px;
+        max-height: 600px;
+        overflow: hidden;
+        background-color: white;
+        border: 1px solid black;
+      }
+      .header {
+        display: flex;
+        justify-content: space-between;
+        padding: 10px;
+      }
+      :host ::slotted([slot='frame-content']) {
+        overflow-y: auto;
+        padding: 10px;
+      }
+    `;
+  }
+
+  get headerNode() {
+    return this.querySelector('[slot="frame-header"]');
+  }
+
+  get bodyNode() {
+    return this.querySelector('[slot="frame-content"]');
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    const frameId = uuid();
+    this.headerNode.setAttribute('id', `frame-header-${frameId}`);
+    this.bodyNode.setAttribute('id', `frame-body-${frameId}`);
+    this.bodyNode.setAttribute('aria-labelledby', `frame-header-${frameId}`);
+    this.bodyNode.setAttribute('aria-describedby', `frame-body-${frameId}`);
+  }
+
+  render() {
+    return html`
+      <h1 class="header">
+        <slot name="frame-header"></slot>
+        <button type="button" @click="${() => this.dispatchEvent(new Event('close-overlay'))}">
+          ⨯
+        </button>
+      </h1>
+      <slot name="frame-content"></slot>
+    `;
+  }
+}
+
+class MyOverlay extends OverlayMixin(LitElement) {
+  _defineOverlayConfig() {
+    return {
+      ...withModalDialogConfig(),
+      contentElementToFocus: this._overlayContentNode?.bodyNode,
+    };
+  }
+
+  render() {
```

Comment:
Can't we render the frame here directly, so this example is really simple? (the external component is tech debt in our extension layer and stems from a time we needed a web component to protect styling (as the contents were moved to the body and now we have the dialog element to point to the top layer)

Reply:
You mean, skip the frame component, and give the overlay 3 slots: invoker, header and content?